src/mlpack/methods/random_forest/random_forest_impl.hpp

```diff
@@ -18,6 +18,23 @@
 namespace mlpack {
 namespace tree {
 
+template<
+    typename FitnessFunction,
+    typename DimensionSelectionType,
+    template<typename> class NumericSplitType,
+    template<typename> class CategoricalSplitType
+>
+RandomForest<
+    FitnessFunction,
+    DimensionSelectionType,
+    NumericSplitType,
+    CategoricalSplitType
+>::RandomForest():
```

Comment:
```suggestion
>::RandomForest() :
```
Super pedantic style comment. :)